### PR TITLE
Added new constructor for #1765 with test case

### PR DESCRIPTION
**Pull request by JustinRChou**
Added new constructor for #1765 with test case.

### CODE REVIEW

File: src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs [hunk 1]
```diff
@@ -185,6 +185,12 @@ public void SourceInAnotherClassPassingSomeDataToConstructor(int n, int d, int q
             Assert.AreEqual(q, n / d);
         }
 
+        [Test, Category("Top"), TestCaseSource("HereIsTheDataWithParametersFor1765", new object[] { 8000, 8, 1000 })]
+        public void SourceInAnotherClassPassingSomeDataToConstructor1765(int n, int d, int q)
```
Comment (CharliePoole) [inline src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs]: The whole point of the issue is to allow use of parameters with data in the same class, specifying only the method name. You've implemented that correctly, but the name of your test says you are testing data in another class. Also, use of the issue number in the test name won't help anyone reviewing the code after we merge it. If you just call it SourceInSameClassPassingSomeDataToConstructor, that tells someone reading the code next year what the test does.

**Comment by CharliePoole** [comment]
That is, ready to merge with the comment fixed.

I find it really annoying this new review facility really annoying. It doesn't match the way we work, with two committers needed to merge and it won't let me edit a comment after I make it. I wonder if it can be turned off?

**Comment by rprouse** [comment]
Looks good to me. Any @nunit/core-team member may merge after the change requested by @CharliePoole is finished.

File: src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs (hunk 1 above)
Comment (JustinRChou) [inline src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs]: Updating the test name and source name.

**Comment by rprouse** [comment]
Looks good, waiting for CI to pass.

**Comment by JustinRChou** [comment]
I have a question about running all the current tests in Visual Studio 2015.
I am using the NUnit test adapters and it doesn't seem to finish.

**Comment by rprouse** [comment]
@JustinRChou The Visual Studio adapter doesn't work well because of the many projects targeting the same frameworks which contain the same tests and test assemblies that aren't intended to be run directly, but are used in the unit tests themselves.

You should build and run the unit tests from the command line using a Visual Studio command prompt. You can do so from the solution root with the command `build.cmd -Target=Test`

**Comment by CharliePoole** [comment]
Unfortunately, timeouts in the OSx build are all too common. I restarted it and it may complete but I'm going ahead with the merge anyway.

**Pull request merged**